.SRCINFO

```diff
@@ -1,21 +1,22 @@
 pkgbase = snapd
 	pkgdesc = Service and tools for management of snap packages.
 	pkgver = 2.31
-	pkgrel = 1
+	pkgrel = 2
 	url = https://github.com/snapcore/snapd
 	install = snapd.install
 	arch = x86_64
 	license = GPL3
 	makedepends = git
 	makedepends = go-pie
 	makedepends = go-tools
+	makedepends = libseccomp
 	makedepends = libcap
-	makedepends = python-docutils
 	makedepends = systemd
 	makedepends = xfsprogs
 	depends = squashfs-tools
 	depends = libseccomp
 	depends = libsystemd
+	optdepends = bash-completion: bash completion support
```

Comment:
Not a strong feeling but does it have to be optional? Is it such a big dependency to avoid?